pkg/kubecost/asset.go

```diff
@@ -2725,6 +2725,10 @@ func (as *AssetSet) ReconciliationMatch(query Asset) (Asset, bool, error) {
 
 	var providerIDMatch Asset
 	for _, asset := range as.assets {
+		// Ignore cloud assets when looking for reconciliation matches
+		if asset.Type() == CloudAssetType {
+			continue;
```

Comment:
Tiniest nit imaginable: I don't think the `;` is necessary.

Reply:
I've been doing to much FE work the last couple weeks